certora/specs/MarketParamsList.spec

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+using Util as Util;
+
+methods {
+    function allocation(MorphoMarketV1Adapter.MarketParams memory marketParams) internal returns (uint256) => ghostAllocation[marketParams.loanToken][marketParams.collateralToken][marketParams.oracle][marketParams.irm][marketParams.lltv];
+
+    function MorphoBalancesLib.expectedSupplyAssets(address morpho, MorphoMarketV1Adapter.MarketParams memory marketParams, address user) internal returns (uint256) => summaryExpectedSupplyAssets(morpho, marketParams, user);
+}
+
+definition max_int256() returns int256 = (2 ^ 255) - 1;
+
+// Would like to do simply one of the following, but it's not supported:
+// mapping (MorphoMarketV1Adapter.MarketParams => uint256)
+// mapping (MorphoMarketV1Adapter.Id => uint256)
+ghost mapping (address => mapping (address => mapping (address => mapping (address => mapping (uint256 => uint256))))) ghostAllocation;
```

Comment:
Would be nice to do simply one of the following, but it's not supported:
- `mapping (MorphoMarketV1Adapter.MarketParams => uint256)`, because struct as key of mappings are not supported in CVL
- `mapping (MorphoMarketV1Adapter.Id => uint256)`, because then to be able to talk about the allocation in a quantified formula you would call `keccak256` and this is not supported in CVL